nostr/event.py

```diff
@@ -57,6 +59,12 @@ def id(self) -> str:
         return Event.compute_id(self.public_key, self.created_at, self.kind, self.tags, self.content)
 
 
+    @property
+    def note_id(self) -> str:
+        converted_bits = bech32.convertbits(unhexlify(self.id), 8, 5)
```

Comment:
nit: instead of `unhexlify`, can we use `bytes.fromhex` to avoid the extra import?